decidim-admin/app/views/decidim/admin/organization/_form.html.erb

```diff
@@ -54,6 +54,12 @@
   </div>
 </div>
 
+<div class="row">
+  <div class="columns xlarge-6">
+    <%= form.check_box :enable_participatory_space_filters %>
```

Comment:
Some i18n is missing to be able to translate this! 😄

Reply:
Worked fine on my computer :smiley: